Website/src/rendering/src/Layout.tsx

```diff
@@ -35,12 +35,19 @@ const Layout = ({ context }: LayoutProps): JSX.Element => {
     context.pageState === LayoutServicePageState.Preview
       ? 'experience-editor-active'
       : '';
+  const contextTitle = context['EventInfo'] as NodeJS.Dict<string | string>;
```

Comment:
Please move that code in its own DEMO TEAM CUSTOMIZATION block with a description of what is the customization.